introduction.md

```diff
@@ -4,3 +4,9 @@ My favorite Code School path is Ruby, although I also enjoy JavaScript.
 
 Introducing myself to the dojo_rules repository.
 I am gjack learning with Codeschool.
+
+Deadly Skills:
+* Ruby
+* Javascript
+* MySQL
+* HTML5

```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?